client/i18n/locales/english/translations.json

```diff
@@ -53,7 +53,7 @@
     "check-code": "Check Your Code (Ctrl + Enter)",
     "check-code-2": "Check Your Code",
     "reset": "Reset",
-    "reset-code": "Reset All Code",
+    "reset-step": "Reset this step",
```

Comment:
```suggestion
    "reset-step": "Reset This Step",
```

Reply:
Completed the above suggested changes to branch.